apps/auth0-rules/package.json

```diff
@@ -11,6 +11,7 @@
     "typecheck": "tsc"
   },
   "dependencies": {
+    "@vedicium/auth0-actions-sdk": "^0.1.3",
```

Comment:
this package doesn't look like it's been maintained, are we sure we don't want to just copy the types we need?

Reply:
I rather not copy paste and maintain our own types if we don't have to.

I doubt Auth0 will release any breaking changes based on our experience thus far.

Why would we copy paste the types?